jenkins/build-and-deploy-birt.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+export remote_command="/usr/bin/ssh -i /keys/infrastructure.pem  ubuntu@birt.eha.io "
+
+#Make sure repo is up to date
+$remote_command "cd /opt/infrastructure && git pull"
+
+#Build the new image
+$remote_command "sudo docker build --no-cache -t birt /opt/infrastructure/docker/images/birt"
+
+#Instantiate the new image
+$remote_command "sudo docker-compose -f /opt/infrastructure/docker/containers/birt.yml up -d"
+
```

Comment:
Good catch! Created a chore to update the Jenkins job that handles this:
https://www.pivotaltracker.com/story/show/135144439